rites/roman1969/src/models/liturgical-day-def.ts

```diff
@@ -81,16 +83,9 @@ export class LiturgicalDayDef implements BaseLiturgicalDayDef {
 
   public get name(): string {
     if (this.#name !== undefined) return this.#name;
-    let name: string;
     // i18nDef from the proper of time already contains the ID
-    if (this.fromCalendarId === PROPER_OF_TIME_NAME) {
-      name = this.#config.i18next.t(this.i18nDef[0], this.i18nDef[1]);
-
-      // i18nDef from general or particular calendars
-    } else {
-      name = this.#config.i18next.t(this.i18nDef[0], this.i18nDef[1]);
-    }
-    this.#name = name;
+    // i18nDef from general or particular calendars
+    this.#name = this.#config.i18next.t(this.i18nDef[0], this.i18nDef[1] as StringMap);
```

Comment:
@emagnier & @tukusejssirs - both results of the if were identical, so I combined them

Reply:
This is really strange in comes from 5dcf4159 where this file was created (`lib/rites/roman/models/liturgical-day-def.ts` was updated and moved into that location) and none of us has noticed this yet. :laughing: :thinking: